mission_number: 4
mission_script: [ [1.79,1.51,1.571,'Grab','Blue'],
                  [0.4,1.78,3.0,'Release','No'],
                  [0.25,0.25,-3.0,'Hold','No']
                ]
# mission_script: [ [1.5,0.5,0.0,'Hold','No'],
#                   [1.5,1.5,0.0,'Hold','No'],
#                   [0.5,1.5,0.0,'Hold','No'],
#                   [0.5,0.5,0.0,'Hold','No'],
#                   [1.5,0.5,0.0,'Hold','No'],
#                   [1.5,1.5,0.0,'Hold','No'],
#                   [0.5,1.5,0.0,'Hold','No'],
#                   [0.5,0.5,0.0,'Hold','No']
#                 ]
Red_ReleasePose: [0.4,1.78,-3.07]
Green_ReleasePose: [0.4,1.55,-3.1]
Blue_ReleasePose: [0.4,1.375,-3.07]
# [1.65,0.55,0.78,'Hold','No'],